app/networkapi/templates/shared/adminsortable_objects_custom.html

```diff
@@ -4,7 +4,7 @@
     <input name="pk" type="hidden" value="{{ object.pk }}" />
     <div id="sortable-object">
         <a href="{% url opts|admin_urlname:'do_sorting' object.model_type_id %}" class="admin_sorting_url">
-            <img class="person-image" src="{{ object.image.url }}">
+            <img class="instance-image" src="{{ object.image.url }}">
```

Comment:
I don't think we should be sharing anything in this file though. Like, some sortable models might not have images so an `img` tag will not be appropriate. This file itself is meant to serve as an html representation of a model/app so it is sorta counter-productive to have it be a general representation of all models/apps since we would need to make all sortable models have certain properties.